control/controlclient/auto.go

```diff
@@ -54,32 +128,38 @@ type Auto struct {
 	logf       logger.Logf
 	expiry     *time.Time
 	closed     bool
+	informCh   chan struct{} // readable when we should inform the server of a change
```

Comment:
oh, sad, I was hoping it'd include the actual change and the informRoutine would track what it's sent

I see now how scary this is